### DISCUSSION TITLE
[Question]how to find a control with an id?

### DISCUSSION THREAD

**Discussion by OmarBelfeki** [flet-dev/flet#4442] [Q&A] [answered]
### Question

how to use page.get_control(id)???
![get_id](https://github.com/user-attachments/assets/7f5f20f3-37db-4b8a-bb4d-75d44dcdaaa6)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Example:
```python
import flet as ft


def main(page: ft.Page):
    x = ft.IconButton(ft.Icons.ADD)
    page.add(x)
    print(type(page.get_control(x.uid)))


ft.app(main)
```

**Comment by OmarBelfeki** [comment]
thank you man you helped me a lot 
===> please add to documentation
